File: Java/프로그래머스/1/155652. 둘만의 암호/둘만의 암호.java
class Solution {
    public String solution(String s, String skip, int index) {
        String answer = "";
        String alphabet = "abcdefghijklmnopqrstuvwxyz";
        String[] skipString = skip.split("");
        for(int i=0; i<skipString.length; i++) {
            alphabet = alphabet.replace(skipString[i],"");
        }
        for(int j=0; j<s.length(); j++) {
               for(int k=0; k<alphabet.length(); k++) {
                   if(s.charAt(j) == alphabet.charAt(k)) {
                       int realIndex = (k+index) % alphabet.length();
                       answer+=alphabet.charAt(realIndex);
                       }
            }
            }
        return answer;
    }}

File: Kotlin/프로그래머스/1/155652. 둘만의 암호/둘만의 암호.kt
class Solution {
    fun solution(s: String, skip: String, index: Int): String {
        var alphabat: String = "abcdefghijklmnopqrstuvwxyz"
        skip.forEach{
            alphabat = alphabat.replace(it.toString(), "")
        }
        return s.map { ch ->
            val i = alphabat.indexOf(ch)
            alphabat[(i + index) % alphabat.length]
        }.joinToString("")
    }
}